src/TraceEvent/TraceEventSession.cs

```diff
@@ -1368,6 +1368,50 @@ public static unsafe List<string> GetActiveSessionNames()
             return activeTraceNames;
         }
 
+        /// <summary>
+        /// Maximum Number of MaxEtwLoggers the system supports
+        /// </summary>
+        private static int? MaxEtwLoggers = null;
```

Comment:
Mentioned in another comment that we can simplify the code by setting `MaxEtwLoggers == 64` here.